Package.swift

```diff
@@ -90,6 +91,7 @@ let package = Package(
         .target(
             name: "SwiftDocCTestUtilities",
             dependencies: [
+                "SwiftDocC",
```

Comment:
question: what is this dependency for?

Reply:
This allows sharing `FileSystemError` implementation from `SwiftDocC` instead of re-declaring it in `SwiftDocCTestUtilities`.